src/server/app.py

```diff
@@ -28,12 +33,12 @@ def submit():
                         status=200,
                         mimetype='application/json')
     except IOError as err:
-        app.logger.error(f'Could not write JSON to file: {err}')
+        app.logger.error('Could not write JSON to file: ')
```

Comment:
The `f` string formatter is heavily supported in python 3. Does this cause you errors when running?

Reply:
Not anymore, added it back!